lsql/judge/views.py

```diff
@@ -475,6 +475,14 @@ def download_ranking(request, collection_id):
             tds = i.find_all("td")
             # Information of a student (Pos, User, Exercises, Score, Solved)
             for j in tds:
+                name = j.find('span', class_='ranking-username')
+                points = j.find('span', class_='ranking-achievement')
```

Comment:
Para el Excel no hace falta que os molestéis en sacar el x1, x2, x3... Con el nombre de usuario es más que suficiente.

Reply:
Aunque si lo dejáis también está bien

Reply:
Bueno al final lo quitamos que metiendo el icono dentro el span también nos da un error.